docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -174,10 +174,86 @@ on open location this_URL
 end open location 
 ```
 
-> **Tip:** To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
-> 
->     do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
-In the Script Editor, save your short script as an “Application Bundle”.
+{% include info title="Tip" content="To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
+    do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
+    In the Script Editor, save your short script as an “Application Bundle”." %}
+
+Find the saved Application Bundle, and Open Contents. Then, open the info.plist file and add the following to the plist dict:
+
+```
+    <key>CFBundleIdentifier</key>
+    <string>com.mycompany.AppleScript.Shotgun</string>
+    <key>CFBundleURLTypes</key>
+    <array>
+      <dict>
+        <key>CFBundleURLName</key>
+        <string>Shotgun</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+          <string>shotgun</string>
+        </array>
+      </dict>
+    </array>
+```
+
+You may want to change the following three strings:
+```
+com.mycompany.AppleScript.{% include product %}
+{% include product %}
+{% include product %}
+```
+
+The third string is the protocol handler; therefore a URL would be:  
+
+```
+shotgrid://something
+```
+
+Then, **delete** the following lines in the `info.plist` file, which fall between the `NSAppleEventsUsageDescription` and `NSSystemAdministrationUsageDescription`:
```

Comment:
Ok, it seems that we are adding a step-by-step process here. I would prefer to use an ordered list for such steps, this makes it way more readable IMO.

Reply:
Followed the format from the rest of the doc, to stay consistent.